network/test/testUtil.go

```diff
@@ -109,7 +109,7 @@ func GenerateNetworks(t *testing.T,
 		state, _ := topology.MockStateForCollectionNodes(t,
 			ids.Filter(filter.HasRole(flow.RoleCollection)), 1)
 		// creates topology instances for the nodes based on their roles
-		tops = GenerateTopologies(t, state, ids, sms, log)
+		tops = GenerateTopologies(t, state, ids, log)
```

Comment:
No longer need `SubscriptionManager` as part of topology, since we input the list of subscribed channels directly to it on `GenerateFanout` (👇  see in files below).